data/moves.ts

```diff
@@ -19247,7 +19247,7 @@ export const Moves: {[moveid: string]: MoveData} = {
 		flags: {protect: 1, mirror: 1, bullet: 1},
 		condition: {
 			noCopy: true,
-			duration: 3,
+			duration: 4,
```

Comment:
This won't work because the client Syrupy status needs to be removed at the end of the third turn.

Reply:
Technically it does work, since in game it does show `Syrupy: 1/4` after your speed has been lowered three times.

Reply:
Yep, and in fact it's relevant because you can't set Syrup Bomb on that final turn. Thanks Game Freak!